src/components/ArticleDetailComments.spec.ts

```diff
@@ -1,14 +1,22 @@
+import { jest } from '@jest/globals'
 import { render, waitFor } from '@testing-library/vue'
 import registerGlobalComponents from 'src/plugins/global-components'
 import { router } from 'src/router'
-import { getCommentsByArticle } from 'src/services/comment/getComments'
-import { deleteComment } from 'src/services/comment/postComment'
 import asyncComponentWrapper from 'src/utils/test/async-component-wrapper'
 import fixtures from 'src/utils/test/fixtures'
-import ArticleDetailComments from './ArticleDetailComments.vue'
 
-jest.mock('src/services/comment/getComments')
-jest.mock('src/services/comment/postComment')
+jest.unstable_mockModule('src/services/comment/getComments', () => ({
+  getCommentsByArticle: jest.fn(),
```

Comment:
Whether I still need to declare the mocked function here? Is it necessary?

Reply:
Yes, it is necessary. The jest mock API for ES modules only accepts a factory function for now, so the exports must be mocked and returned from that factory function.